insights/terraform/projects/bb2/services/quicksight/README.md

```diff
@@ -0,0 +1,627 @@
+                                    }
+                                }
+
+                                # (3 unchanged blocks hidden)
+                            }
+
+                            # (3 unchanged blocks hidden)
+                        }
+
+                        # (2 unchanged blocks hidden)
+                    }
+                }
+
+                # (3 unchanged blocks hidden)
+            }
+          ~ sheets {
+                name         = "PROD Applications Token Activity"
+                # (2 unchanged attributes hidden)
+
+              ~ visuals {
+                  ~ pivot_table_visual {
+                        # (1 unchanged attribute hidden)
+
+                      ~ chart_configuration {
+                          ~ table_options {
+                                # (3 unchanged attributes hidden)
+
+                              - row_field_names_style {
+                                  - height    = 40 -> null
+                                  - text_wrap = "WRAP" -> null
+                                }
+
+                              ~ row_header_style {
+                                  ~ height                    = 0 -> 40
+                                  + text_wrap                 = "WRAP"
+                                    # (2 unchanged attributes hidden)
+
+                                    # (1 unchanged block hidden)
+                                }
+
+                                # (2 unchanged blocks hidden)
+                            }
+                          ~ total_options {
+                              ~ row_subtotal_options {
+                                    # (1 unchanged attribute hidden)
+
+                                  - total_cell_style {
+                                    }
+                                }
+
+                                # (3 unchanged blocks hidden)
+                            }
+
+                            # (3 unchanged blocks hidden)
+                        }
+
+                        # (2 unchanged blocks hidden)
+                    }
+                }
+
+                # (5 unchanged blocks hidden)
+            }
+
+            # (86 unchanged blocks hidden)
+        }
+
+        # (2 unchanged blocks hidden)
+    }
+```
+
+## Summary of Analysis Terraform Issues
+
+The following is a description of some issues related to the plan changes **that always show** in both analyses plan outputs. Even after a successful terraform APPLY.
+
+- Some types of `height` values are set to 0 internally; however, setting this in the TF code produces an error during the apply, with a message that the value must be greater than 0.
+- Some types of `height` values do not change (Ex. height=40 for row_header_style).
+- Some `text_wrap = "WRAP"` values do not change.
+- The `column_configurations` shown will flip positions, when matching them in the TF code and after a new apply.
+- The theme setting does not work.
+
```

Comment:
@dtisza1 i think there's also something about the dimension / measure that pops up as well

Reply:
> @dtisza1 i think there's also something about the dimension / measure that pops up as well

@meliGuzman I added some more info for that `column_configurations` . Does that read better?